config/src/main/java/org/opennms/integration/api/xml/schema/eventconf/EventLabelComparator.java

```diff
@@ -32,11 +32,11 @@
 import java.util.Comparator;
 
 public class EventLabelComparator implements Comparator<Event>, Serializable {
-    private static final long serialVersionUID = 7976730920523203921L;
+    private static final long serialVersionUID = 1L;
 
     @Override
     public int compare(final Event e1, final Event e2) {
-        if (e1.getEventLabel() == e2.getEventLabel()) return 0;
+        if (e1.getEventLabel().equals(e2.getEventLabel())) return 0;
```

Comment:
The null checks should be placed before the `equals` check.

Reply:
Looks like Sonar caught another issue which needs to be fixed ... if only Java had an optional chaining operator...

Reply:
It sure did! Ergh. Good for you, Sonar.